### PR TITLE
Qt: Misc Fixes

### PR DESCRIPTION
**Pull request by kamfretoz**
This PR adds in fixes mainly to the controller setting's virtual controller list, making it slightly wider and the text word wrapped to account for localization.

See commits for list of changes.

Before:
Screenshot_20240910_201558

After:
Screenshot_20240910_201549

### CODE REVIEW

**Comment by Mijyuoon** [comment]
Doesn't the process of making a PR like this technically violate the new license's clause about not being allowed to make modifications?

**Comment by stenzek** [comment]
I have no problem with contributors forking for the purpose of contributing, regardless of the license conditions. In fact, I encourage it, since this was one of the reasons I switched in the first place - preventing fragmentation of the community due to people forking, and it ending up unmaintained.

Also see https://docs.github.com/en/site-policy/github-terms/github-terms-of-service#5-license-grant-to-other-users.

**Comment by detuur** [comment]
@stenzek you are not allowed to relicense other contributors' code away from GPL. If you wish to relicense your code base, for every line of code that was not written by you, you need to either remove their code from your code base altogether, or receive written permission from the authors to relicense their code.

The GPL's defining feature is that it requires derivative work (i.e. any commit after their code has been merged) to be licensed under the same license. The project does not seem to have a CLA (Contributor's License Agreement) that transfers ownership of their code or that pre-approves license changes.

tl;dr: Your relicensing is in violation of the GPL.

**Comment by stenzek** [comment]
I am well aware of how licenses work. That's why I changed, because others blatantly violated both the GPL, by not attributing and stripping my copyright.

I have approval of prior contributors, and if I did somehow miss you, then please advise me so I can rewrite that code.
